src/components/Task_list/Calendar.jsx

```diff
@@ -12,9 +12,9 @@ function TaskCalendar() {
   const [events, setEvents] = useState([
     {
       id: 1,
-      title: 'Task 1',
-      start: new Date(2024, 3, 7, 10, 0),
-      end: new Date(2024, 3, 7, 12, 0),
+      title: 'Task asfkljsa;djf;lasdjf;ladsjfl;dasjf',
```

Comment:
Is this hard coded test data? If so then perhaps it should be added to the database/seed.js file so that it gets added to the database then the Calendar component can retrieve it via an axios request.